templates/php/base/requests/api.twig

```diff
@@ -0,0 +1,8 @@
+        return $this->client->call(Client::METHOD_{{ method.method | caseUpper }}, $path, [
```

Comment:
I noticed some files does not start on the left, is that on purpose?

Reply:
yes to maintain the formatting on the generated SDK